src/transport_ofi.c

```diff
@@ -718,7 +718,7 @@ static inline int query_for_fabric(struct fi_info ** p_info, char *provname)
     ep_attr.type              = FI_EP_RDM; /* reliable connectionless */
     hints.fabric_attr	      = &fabric_attr;
     tx_attr.op_flags          = FI_DELIVERY_COMPLETE;
-    tx_attr.inject_size       = shmem_transport_ofi_max_buffered_send;
+    tx_attr.inject_size       = shmem_transport_ofi_max_buffered_send; /*require provider to support this as a min*/
```

Comment:
essentially this is telling the provider you *must* support at least this inject size, the provider will check if their supported inject is at least this size, if it is less than the requested inject size it will abort saying that it cannot support your requirements. Thus we should treat it as a minimal allowable requirement